### ISSUE TITLE
Python to give back to IOS

### ISSUE THREAD

**Issue by knsjoon** [jc211/NeRFCapture#16]
I saw this github while searching for ways to send arkit data to my ubuntu machine. 
I think your app / code does just that: sending live streams of RGB-D images and pose. 

Since I am working on a project that needs to give the user a feedback like 3D bounding box around objects (I am running a 3D segmentation using machine learning on ubuntu) and show it on Iphone, I wonder if cyclonedds or flask can be used/mixed to fulfill this. 

Thank you!

**Comment by jc211** [comment]
Yes absolutely. I'm not sure what flask is, but cyclonedds will do the trick. You will need a dds subscriber in the app and a publisher in your ubuntu machine with a message between them that has your 3D bounding box. You'll have to intercept that message in the app and draw it in the view.
